doc/source/tutorials/local/Analysing_data/Working_with_co2.rst

```diff
@@ -0,0 +1,134 @@
+
+First we'll add some surface measurment data
+
+.. code:: ipython3
+
+    populate_surface_data()
+
+Next we'll add some flux data.
+
+.. code:: ipython3
+
+    populate_flux_data()
+
+And finally some footprint data
+
+.. code:: ipython3
+
+    populate_footprint_data()
+
+2. Creating a model scenario
+----------------------------
+
+To link together our observations to our ancillary data we can create a
+``ModelScenario`` object, as shown in the previous tutorial
+`3_Comparing_with_emissions.ipynb <3_Comparing_with_emissions.ipynb>`__,
+using suitable keywords to grab the data from the object store.
+
+.. code:: ipython3
+
+    from openghg.analyse import ModelScenario
+
+    site = "TAC"
+    domain = "EUROPE"
+    species = "co2"
+    height = "185m"
+    source_natural = "natural"
+    start_date = "2017-07-01"
+    end_date = "2017-07-07"
+
+    scenario = ModelScenario(site=site,
+                             inlet=height,
+                             domain=domain,
+                             species=species,
+                             source=source_natural,
+                             start_date=start_date,
+                             end_date=end_date)
+
+We can plot our observation timeseries using the
+``ModelScenario.plot_timeseries()`` method as before:
+
+.. code:: ipython3
+
+    scenario.plot_timeseries()
+
+We can also check trace details of the extracted data by checking the
+available metadata. For instance for our footprint data we would expect
+this to have an associated species and for this to be labelled as “co2”:
+
+.. code:: ipython3
+
+    footprint_metadata = scenario.footprint.metadata
+    footprint_species = footprint_metadata["species"]
+    print(f"Our linked footprint has an associated species of '{footprint_species}'")
+
+3. Comparing data sources
+-------------------------
+
+Once the correct high frequency emissions and footprints have been
+linked for our carbon dioxide data, we can start to plot comparisons
+between the sources and our measurement data.
+
+.. code:: ipython3
+
+    scenario.plot_comparison(baseline="percentile")
+
+As in the previous tutorial, multiple fluxes can be linked to your
+``ModelScenario`` object if required. This can include additional high
+frequency (<24 hourly) or low frequency flux data. In this case we have
+added monthly “fossil fuel” emissions:
+
+.. code:: ipython3
+
+    source_fossil = "ff-edgar-bp"
+
+    scenario.add_flux(species=species,
+                      source=source_fossil,
+                      domain=domain)
+
+.. code:: ipython3
+
+    fossil_flux = scenario.fluxes[source_fossil]
+    fossil_flux
+
+If we plot the modelled measurement comparison, this will stack the
+natural and fossil fuel flux sources and combine with the footprint data
+in an appropriate way:
+
+.. code:: ipython3
+
+    # scenario.plot_comparison(baseline="percentile", recalculate=True)
```

Comment:
*(Same comment as above)* Does clear_tutorial_store() need to be added here (at least for consistency)?